### DISCUSSION TITLE
Version 7.44.0-rc.3

### DISCUSSION THREAD

**Discussion by bluebill1049** [react-hook-form/react-hook-form#10380] [General]
🐞 fix #9037 bugs that occur in the presence of Array polyfills (#10328)
🚔 added TTransformedValues to FormProvider (#10368)

```
FormProviderProps<TFieldValues, TContext, TTransformedValues>
```

🫥 close #10348 stop shouldUseNativeValidation pass down constraint props (#10350)

thanks to @yasamoka & @Torvin

<hr /><em>This discussion was created from the release <a href='https://github.com/react-hook-form/react-hook-form/releases/tag/v7.44.0-rc.3'>Version 7.44.0-rc.3</a>.</em>

**Comment by kasparkallas** [comment]
`TransformedValues` is not passed forward to `UseFormReturn`. (Nit-pick: "TransformedValues" should be "TTransformedValues".)

```ts
export const useFormContext = <
  TFieldValues extends FieldValues,
  TransformedValues extends FieldValues | undefined = undefined,
>(): UseFormReturn<TFieldValues> =>
  React.useContext(HookFormContext) as UseFormReturn<
    TFieldValues,
    TransformedValues
  >;
```

For code symmetry and future-proofness, it probably makes sense to have TContext there as well, e.g. `<TFieldValues, TContext, TTransformedValues>`.
